images/html-editor/list-ordered.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18" mirror-rtl="true">
   <path fill="#72777a" d="M2 .5v3a.5.5 0 0 1-1 0V1.31l-.28.14a.54.54 0 0 1-.22.05.5.5 0 0 1-.45-.28.506.506 0 0 1 .23-.67l1-.5a.474.474 0 0 1 .35-.03.514.514 0 0 1 .13.05.02.02 0 0 0 .02.01c.03.03.06.05.09.08a.864.864 0 0 1 .08.12A.54.54 0 0 1 2 .5zM3 14.5v3a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1H2v-.5h-.5a.5.5 0 0 1 0-1H2V15H.5a.5.5 0 0 1 0-1h2a.5.5 0 0 1 .5.5zM17 4H6a1 1 0 0 1 0-2h11a1 1 0 0 1 0 2zM17 11H6a1 1 0 0 1 0-2h11a1 1 0 0 1 0 2zM17 18H6a1 1 0 0 1 0-2h11a1 1 0 0 1 0 2zM3 10.5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1-.5-.5.476.476 0 0 1 .09-.29s.05-.07.06-.09l.59-.84s.67-.94.68-.95a.337.337 0 0 0 .08-.17.247.247 0 0 0-.25-.27.235.235 0 0 0-.2.1c-.032.05-.043.11-.076.153a.5.5 0 0 1-.414.217.5.5 0 0 1-.48-.63A1.232 1.232 0 0 1 .74 7 1.25 1.25 0 0 1 2.5 8.14a1.29 1.29 0 0 1-.16.61c-.06.09-.87 1.25-.88 1.25H2.5a.5.5 0 0 1 .5.5z"/>
```

Comment:
Yeah so this one and the checkbox-list one fall into a category where we should be mirroring but NOT mirroring the numbers or the checkboxes themselves. That means we can't do them programatically. My thinking was to just mirror them for now and figure out an alternate method later if we deem it high enough priority. I can do the opposite though -- not mirror them.